hp_list/main.c

```diff
@@ -195,7 +291,7 @@ list_node_t *list_node_new(list_key_t key)
     list_node_t *node = aligned_alloc(128, sizeof(*node));
     assert(node);
     *node = (list_node_t){.magic = LIST_MAGIC, .key = key};
-    (void) atomic_fetch_add(&inserts, 1);
```

Comment:
You changed the behavior, which should be forbidden.

Reply:
You have to ensure the invocation of `atomic_fetch_add` when `RUNTIME_STAT` is not defined.

Reply:
`inserts` and `deletes` are declared with `RUNTIME_STAT`, `atomic_fetch_add(&inserts, 1);` will cause`inserts undeclared`.

Reply:
It is time to consolidate the use of these variables. Simplicity is usually the best principle.